xmltest.cpp

```diff
@@ -1641,6 +1641,20 @@ int main( int argc, const char ** argv )
 		}
 	}
 
+	{
+		// Oh those memory leaks.
+		// Only way to see these is in the (Windows) allocator tracking.
+		{
+			XMLDocument doc;
+			doc.NewElement("LEAK 1");

```

Comment:
The element name is therefore misleading - should be something like "SHOULD NOT BE LEAKED 1".